src/ObservableOption.ts

```diff
@@ -0,0 +1,417 @@
+/**
+ * @since 0.6.14
+ */
+import { Alt1 } from 'fp-ts/lib/Alt'
+import { Applicative1 } from 'fp-ts/lib/Applicative'
+import { Apply1 } from 'fp-ts/lib/Apply'
+import { flow, identity, Predicate, Refinement } from 'fp-ts/lib/function'
+import { Functor1 } from 'fp-ts/lib/Functor'
+import { IO } from 'fp-ts/lib/IO'
+import { Monad1 } from 'fp-ts/lib/Monad'
+import { MonadIO1 } from 'fp-ts/lib/MonadIO'
+import { MonadTask1 } from 'fp-ts/lib/MonadTask'
+import * as O from 'fp-ts/lib/Option'
+import { pipe } from 'fp-ts/lib/pipeable'
+import { Observable } from 'rxjs'
+import { catchError } from 'rxjs/operators'
+import { MonadObservable1 } from './MonadObservable'
+import * as R from './Observable'
+
+// -------------------------------------------------------------------------------------
+// model
+// -------------------------------------------------------------------------------------
+
+/**
+ * @category model
+ * @since 0.6.14
+ */
+export interface ObservableOption<A> extends Observable<O.Option<A>> {}
+
+// -------------------------------------------------------------------------------------
+// constructors
+// -------------------------------------------------------------------------------------
+
+/**
+ * @category constructors
+ * @since 0.6.14
+ */
+export const none: ObservableOption<never> =
+  /*#__PURE__*/
+  pipe(O.none, R.of)
+
+/**
+ * @category constructors
+ * @since 0.6.14
+ */
+export const some: <A = never>(a: A) => ObservableOption<A> =
+  /*#__PURE__*/
+  flow(O.some, R.of)
+
+/**
+ * @category constructors
+ * @since 0.6.14
+ */
+export const someObservable: <A = never>(ma: Observable<A>) => ObservableOption<A> =
```

Comment:
`someObservable` and `fromObservable` are aliases, I would only keep `fromObservable`